qiskit/algorithms/optimizers/scipy_minimizer.py

```diff
@@ -0,0 +1,151 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2018, 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Wrapper class of scipy.optimize.minimize."""
+
+from collections.abc import Sequence
+from typing import Any, Callable, Dict, Optional, Union
+
+import numpy as np
+from scipy.optimize import Bounds, minimize
+
+from qiskit.utils.validation import validate_min
+
+from .optimizer import Optimizer, OptimizerSupportLevel
+
+
+class ScipyMinimizer(Optimizer):
```

Comment:
Could we call this `SciPy` with a capital `P` to be consistent with the many places where we have `NumPySomething`? 🙂 
```suggestion
class SciPyMinimizer(Optimizer):
```

Reply:
Thanks. As for the name, I'm not sure which is better: SciPyMinimizer or SciPyOptimizer? What do you think?

Reply:
I'd go for `ScipPyOptimizer` since it's closer to the name of the parent class. And during the VQE & Optimizer refactor we'll change the name of the `optimize` method to `minimize`, then it's also clear that all the optimizers minimize 🙂

Reply:
Thanks. It makes sense. I'll change the name to `SciPyOptimizer`.